addons/web/static/src/js/core/dom.js

```diff
@@ -548,6 +548,10 @@ var dom = {
      * @return {Promise}
      */
     scrollTo(el, options = {}) {
+        // A missing el freezes the browser during the call to jQuery's closestScrollable.
+        if (!el) {
+            return;
+        }
```

Comment:
Mmh, I'd prefer two things instead of this:
- `closestScrollable` can never loop (to respect the convention of any jquery function: `$().xxx()` -> `false` or `$()`
- `scrollTo` is never called with `el === undefined` (the most important is this one, makes me think your diff here is just hiding a problem no? :grimacing:)

Reply:
Well given the freezing nature of the problem, I prefer a no-op in case this happens again in the future - or I could raise an Error.
The `start()` of link_tools did call `scrollTo` with `this.$(':visible:last')[0]` which led to `undefined` if the link focus was lost - this is what I hope to have fixed in the second commit.

Reply:
Ok :+1: Let's raise an error :+1: